app/models/comment.rb

```diff
@@ -97,6 +95,14 @@ def self.tree_for(commentable, limit = 0)
       .to_h
   end
 
+  def self.title_deleted
```

Comment:
Should these be private instance methods? They are defined on the class level but only called on the instance level. I understand that conceptually they aren't instance methods but it would make the call-sites a bit less awkward since we can skip the `self.class` bit.

Reply:
Sounds good.

Reply:
Sorry, during spec resolving I have found app/views/api/v0/comments/_comment.json.jbuilder actually calls this value. Then I think it might be better to leave it as is for a slight win on performance.